README.md

```diff
@@ -446,6 +446,20 @@ var showdown    = require('showdown'),
     converter = new showdown.Converter({ extensions: ['myExtension'] });
 ```
 
+## Building
+
+Building your clone of the repository is easy.
+> Prerequesites: [npm](https://www.npmjs.com/package/npm) and [npx](https://www.npmjs.com/package/npx) must be installed.
+
+1. run `npm install`: installs all the needed third-party packages into the [node_modules](/node_modules)  folder, based on the contents of [package-lock.json](/package-lock.json) and [package.json](/package.json).
```

Comment:
```suggestion
1. run `npm install`.
```
`npm install` doesn't do any magic here, and if you are talking about developers as the target audience for this section, we can safely omit this kinda  excessive description.